examples/universal-cnf/vppagent/pkg/ucnf/ucnf_nse.go

```diff
@@ -39,8 +119,33 @@ func NewUcnfNse(configPath string, verify bool, backend config.UniversalCNFBacke
 	}
 
 	configuration := common.FromEnv()
+	newIpamService := func(addr string) config.IpamService {
+		conn, err := grpc.Dial(addr, grpc.WithInsecure())
+		if err != nil {
+			logrus.Fatal("unable to connect to ipam server: %v", err)
+		}
 
-	pe := config.NewProcessEndpoints(backend, cnfConfig.Endpoints, configuration, ceAddons)
+		ipamAllocator := ipprovider.NewAllocatorClient(conn)
+		ipamService := ipamService{
+			IpamAllocator:     ipamAllocator,
+			registeredSubnets: make(chan *ipprovider.Subnet),
+			ctx:               ctx,
+		}
+		go func() {
+			logrus.Info("begin the renew process")
```

Comment:
Add some information here, looking at the logs, does not tell me what exactly am I renewing.

Reply:
done